src/main/java/org/ow2/proactive/procci/model/occi/metamodel/Category.java

```diff
@@ -41,20 +41,19 @@
 import com.google.common.collect.ImmutableSet;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import lombok.Setter;
 
 /**
  * Category is the basis type of identification mecanism
  **/
 @EqualsAndHashCode(of = { "scheme", "term" })
+@Getter
```

Comment:
you can use simply @Data of lombok, so all final field, it will generate only getter method, and setter method in addition for no-final field